sklearnex/utils/tests/test_validation.py

```diff
@@ -0,0 +1,236 @@
+# ==============================================================================
+# Copyright 2024 UXL Foundation Contributors
```

Comment:
@napetrov I found only a similar example with a UXL copyright in the oneapi-spec repo in uxlfoundation, let me know if this isn't sufficient.